workers/data_refinery_workers/processors/smashing_utils.py

```diff
@@ -74,6 +76,10 @@ def prepare_files(job_context: Dict) -> Dict:
                 smashable_files = smashable_files + [(smashable_file, sample)]
                 seen_files.add(smashable_file)
                 found_files = True
+            else:
+                job_context['filtered_samples'][sample.accession_code] = {
+                    'reason': 'We could not find a valid smashable file for this sample.'
```

Comment:
We shouldn't use `smashable` in copy destined for user consumption. Maybe `This sample did not have a processed file associated with it in our database.` would be better?